src/Http/Requests/UpdateContactRequest.php

```diff
@@ -29,8 +29,9 @@ public function rules(): array
                 'nullable',
                 (new ExistsWithIgnore('price_lists', 'id'))->whereNull('deleted_at'),
             ],
-            'customer_number' => 'sometimes|string',
-            'creditor_number' => 'string|nullable',
+            'customer_number' => 'sometimes|string|unique:contacts,customer_number',
+            'creditor_number' => 'string|nullable|unique:contacts,creditor_number',
+            'debtor_number' => 'string|nullable|unique:contacts,debtor_number',
```

Comment:
validate unique in UpdateContact action, because the updated record should be excluded from unique check.
(affects creditor_number and debtor_number)